.github/workflows/translations-checker.yml

```
@@ -0,0 +1,35 @@
+name: "Find missing translations"
+
+on:
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  translations:
+    name: Translations Checker
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@5a4ac9002d0be2fb38bd78e4b4dbde5606d7042f
+      - uses: actions/setup-node@46071b5c7a2e0c34e49c3cb8a0e792e86e18d5ea
+        with:
+          node-version: 14
+      - run: npm ci
+      - name: Find missing languages
+        id: find_string
+        run: npm run translation:missing
+      - name: Read missingTranslations.json
+        id: translations
+        uses: juliangruber/read-file-action@74dc4b4435b60f96f84bcba916659c779f46ba0a
+        with:
+          path: ./missingTranslations.json
+      - uses: marocchino/sticky-pull-request-comment@322a2451dae6c7831d1a8b931275a7f78147c888
+        with:
+          GITHUB_TOKEN: ${{ secrets.DEFICHAIN_BOT_GITHUB_TOKEN }}
+          header: Missing Translations
+          recreate: true
+          message: |
+            ## Translations Report
+            ***
+            ```javascript
+            ${{ steps.translations.outputs.content }}
+            ```
```

```suggestion
            ### Missing Translations Report
            
            The following translations are missing for this pull request.
            
            ```json
            ${{ steps.translations.outputs.content }}
            ```
```